src/pages/filter/ui/panels/function-panel/function-panel.store.ts

```diff
@@ -64,8 +89,32 @@ class FunctionPanelStore {
     }
   }
 
-  public fetchStatFunc(componentName: string, params: string): void {
-    filterStore.fetchStatFuncAsync(componentName, params)
+  public fetchStatFunc(componentName: string, params: string) {
+    return filterStore.fetchStatFuncAsync(componentName, params)
+  }
+
+  public getStringScenario(arrayScenario: TScenario[]): string {
```

Comment:
If you do some adaptations and normalizations, better to move them in utils functions (since we don't have adapters so far). First, they might be reusable, second, store is not supposed to do "dirty work". It should be imperative so declarative lower-level things must be outside. The idea is store **controls** the business logic, not tells what to do in particular